<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
border: 2px solid rgb(0, 0, 0);
}

QPushButton {
 color: #fff;
font: bold;
 border: 2px solid rgb(42,60,82);
 border-radius:5px;
	background-color: rgb(134, 134, 134);
 }

QPushButton:checked {
 color: #000;
font: bold;
 border: 2px solid #BDBDBD;
 border-radius:5px;
	background-color: rgb(179, 179, 179);
 }</string>
  </property>
  <widget class="QPushButton" name="dialog_yes">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>예</string>
   </property>
  </widget>
  <widget class="QLabel" name="dialog_text">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>카드인식 or 사원번호를
입력해주세요?</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="dialog_icon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../image.qrc">:/new/prefix1/image/how.PNG</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="dialog_no">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>아니오</string>
   </property>
  </widget>
  <widget class="QLabel" name="dialog_info">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>451</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dialog_yes</sender>
   <signal>clicked()</signal>
   <receiver>dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialog_no</sender>
   <signal>clicked()</signal>
   <receiver>dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
